providers-sdk/v1/lr/cli/cmd/markdown.go

```diff
@@ -70,22 +70,18 @@ var markdownCmd = &cobra.Command{
 
 		var lrDocsData docs.LrDocs
 		docsFilepath, _ := cmd.Flags().GetString("docs-file")
-		_, err = os.Stat(docsFilepath)
-		if err == nil {
-			content, err := os.ReadFile(docsFilepath)
-			if err != nil {
-				log.Fatal().Err(err).Msg("could not read file " + docsFilepath)
-			}
-			err = yaml.Unmarshal(content, &lrDocsData)
-			if err != nil {
-				log.Fatal().Err(err).Msg("could not load yaml data")
-			}
 
-			log.Info().Int("resources", len(lrDocsData.Resources)).Msg("loaded docs from " + docsFilepath)
-		} else {
-			log.Info().Msg("no docs file provided")
+		content, err := os.ReadFile(docsFilepath)
```

Comment:
Instead of making this required, we should change the check logic. If a path is provided, we check that it exists. So we only load the yaml when the path is != ""

Reply:
I updated the code to ensure we do not silently ignore a file that was provided with a wrong path. The current logic still treats the yaml as optional since it is not required to generate markdown.